src/haptics.cpp

```diff
@@ -21,42 +56,37 @@ init_haptics() {
  */
 void actuate_motor(int motor_pin, int duration, double percent_motor) {
   digitalWrite(MOTOR_PINS[motor_pin] + EN_OFFSET, HIGH);  // enable
-  int frequency = 3;
+  int half_period = 3;
   if (percent_motor > 0.85) {
     Serial.println("Motor percentage above 85%. Using 85%");
     percent_motor = 0.85;
   }
-  int numCycles = duration / (frequency * 2);
+  percent_motor = 1;
```

Comment:
why does percent_motor immediately get set to 1 here? don't we want to keep it below 0.85?